wire-ios-sync-engine/Source/Synchronization/ZMOperationLoop+PushChannel.swift

```diff
@@ -60,7 +60,7 @@ extension ZMOperationLoop: ZMPushChannelConsumer {
             } else {
                 WaitingGroupTask(context: syncMOC) {
                     do {
-                        try await self.updateEventProcessor.processEvents(events)
+                        try await self.updateEventProcessor.processLiveEvents(events)
```

Comment:
question: so the point here is that we don't post the notifications for live events?

Reply:
well, liveEvents were not blocking sending messages (only the quick sync), so that was my thinking to don't block here